### PR TITLE
(PUP-6961) Add risk tag to acceptance tests

### PR DESCRIPTION
**Pull request by johnduarte**
Add a beaker tag to all of the existing acceptance
tests to reflect their assessed risk level. These tags will be used
to assist with tiering the execution of the test suite in order
to provide faster test suite execution.

### CODE REVIEW

**Comment by johnduarte** [comment]
@MosesMendoza I am interested in your thoughts regarding tests that can be tagged as low risk.

**Comment by Iristyle** [comment]
Wow - do we have a guide / doc somewhere that states how the levels were selected?

**Comment by johnduarte** [comment]
@Iristyle creating a doc is a great idea. We do not currently have one. Basic heuristics for this PR were:

1. Default to medium
1. Define existing per commit tests as high
1. Apply low to zfs and zone tests that pertain to a smaller subset of our supported matrix.

**Comment by MosesMendoza** [comment]
@johnduarte thoughts on adding that information to a README? i.e., maybe `acceptance/README.md` (does not exist yet)

**Comment by johnduarte** [comment]
@MosesMendoza I have added information about the tags to the README.md. Let me know if more detail should be incorporated.

File: acceptance/README.md [hunk 1]
```diff
@@ -145,6 +145,33 @@ There also may be scenarios where you want to specify the host(s) to release. E.
...
     HOST_NAMES=lvwwr9tdplg351u,ylrqjh5l6xvym4t bundle exec rake ci:release_hosts
 
+### Running tagged tests
+
+Tests runs can be segmented by tags defined in the tests themselves. These
+tags have been standardized into the following.
+
+* risk:high
+* risk:medium
+* risk:low
```
Comment (johnduarte) [inline acceptance/README.md]: Great questing @glennsarti . I have amended the README to clarify that untagged tests will be run weekly.

Comment (glennsarti) [inline acceptance/README.md]: Hrmmm.... I would argue that untagged tests are high risk as;
1 - Having no tag (probably) shows no thought has gone into how risky they are, therefore we should run them as often as possible to figure out how risky they really are
2  - It's far too easy to forget to tag a test and then only find out  many days later that it failed

Comment (johnduarte) [inline acceptance/README.md]: @glennsarti you have brought up a requirement that @MosesMendoza called to my attention as well. Your concern about new tests will be addressed by adding any tests that have changes to them dynamically to the per commit runs.

I would argue against defaulting untagged tests to the per commit runs. The purpose of the tiering workflow is to make sure the cost of execution for frequent CI runs remains as low as possible. I agree that there is risk in having untagged tests, but most new tests should not jump into the per commit runs.

Automatically running changed (which would include new) tests in the per commit run should mitigate this risk. I am also interested in your thoughts about adding a commit hook to ensure that committed tests contain a tag.

Comment (johnduarte) [inline acceptance/README.md]: As it turns out, the way that Beaker implements tag filtering a test without any tags is not filtered out when using the '--tag' option facility. Therefore, untagged tests will always be run when using the '--tag' option.
@glennsarti , this behavior addresses your concern about frequently running untagged tests.
I have updated the README to reflect this.

**Pull request closed without merging**